packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -905,13 +938,13 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 	public getPendingBlobs(): IPendingBlobs {
 		const blobs = {};
 		for (const [key, entry] of this.pendingBlobs) {
-			blobs[key] = entry.minTTLInSeconds
-				? {
-						blob: bufferToString(entry.blob, "base64"),
-						uploadTime: entry.uploadTime,
-						minTTLInSeconds: entry.minTTLInSeconds,
-				  }
-				: { blob: bufferToString(entry.blob, "base64") };
+			blobs[key] = {
+				blob: bufferToString(entry.blob, "base64"),
+				attached: entry.attached,
+				acked: entry.acked,
+				...(entry.minTTLInSeconds && { minTTLInSeconds: entry.minTTLInSeconds }),
```

Comment:
i don't understand what this is doing. i believe this will rest expand a boolean into the entry which seems wrong